http_test.go

```diff
@@ -194,7 +194,7 @@ func TestHandlers(t *testing.T) {
 	var reputations []Reputation
 	err = json.Unmarshal(buf, &reputations)
 	assert.Nil(t, err)
-	assert.Equal(t, 5, len(reputations))
+	assert.Equal(t, 6, len(reputations))
```

Comment:
Why is this now `6`?

Reply:
added a sample email reputation entry in the base test

Reply:
Aw I see it now, thanks.